src/client/components/AdminPanel/AdminListingsToggle.js

```diff
@@ -34,9 +41,9 @@ export const AdminListingsToggle = ({ shortname, listings, mediaType }) => {
   // Summary includes the shortname and the number of listings
   // If missing shortname, include an orange warning pill
   const shortnameSummary = (
-    <>
+    <View>
```

Comment:
I figured <> </> is better than wrapping with a < View >. I'll create a ticket to get this change done (change all <View> without props to <>)

Reply:
What's the reason <></> is better than <View>?

Reply:
So <View> renders a lot of unnecessary html components.
See https://www.barrymichaeldoyle.com/fragment/